### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by tilgovi** [openannotation/annotator#496]
This issue is here as an umbrella for accessibility work.

Some good work has gone into some plugins for the EdX platform. I would like to see us start rolling some of these changes into core eventually.

Here's an email thread that should be a good jumping off point for discovering all that's been happening over there: https://groups.google.com/a/list.hypothes.is/d/msgid/dev/550C8A37.4000303%40w3.org

Follow your nose.

**Comment by manoj-mass** [comment]
Hi @tilgovi , 

Is it possible to use `<mark>` tags instead of `<span>` tags to do the highlighting, by doing so screen reader tools like VoiceOver, JAWS, NVDA will be automatically identify the highlighted text area.

`<mark>WCAG stands for Web Content Accessibility Guidelines. </mark>`

VoiceOver macOS Speaks ", Start Highlight, " before and ", End Highlight, " after the highlight.
* Works on iOS and macOS VoiceOver, JAWS, NVDA

**Comment by tilgovi** [comment]
@manoj-mass you would have to create your own highlighter implementation for that.
